src/database/Database.cpp

```diff
@@ -237,6 +237,9 @@ Database::applySchemaUpgrade(unsigned long vers)
         mApp.getPersistentState().setRebuildForType(CONTRACT_DATA);
         mApp.getPersistentState().setRebuildForType(CONTRACT_CODE);
         break;
+    case 21:
```

Comment:
why not 20?

Reply:
Because testnet is already on 20 without the expiration table.